### ISSUE TITLE
kitti test do not have test labels

### ISSUE THREAD

**Issue by MisayaZ** [manutdzou/KITTI_SSD#3]
Hi ,Thank you for your contribution. I have a problem that kitti datasets does not have test label, how do you generate test lmdb?

**Comment by ysc703** [comment]
@MisayaZ  Have you found the test label files?

**Comment by wangxiaoyaner** [comment]
Labels for test set is not available. You can divide the train set into train set and validation set and use validation set as test set.

**Comment by IamMegatron2025** [comment]
I am also curious about this

**Comment by rdabane** [comment]
I'm looking for test labels as well. Have they been made available?

**Comment by annonp** [comment]
@wangxiaoyaner Them, what do we do about the `testing` folder of the KITTI dataset? Is it for just submitting the predicted results like in competition?

Edit: On a second thought, maybe it is only for submitting the predicted results?
